const axios = require('axios')


const getDetail = async () =>{
    try {
        
    } catch (error) {
        console.log(error)
        return error
    }
}